app/views/articles/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<style>
+  a{
+      margin :2px;
+  }
+
+</style>
+
+
+<h1>List of all articles</h1>
+
+
+<table border="1">
+<tr>
+  <td><strong>Title</strong></td>
+  <td><strong>Tag</strong></td>
+</tr>
+<% @articles.each do |article|%>
+    <tr>
+<td><a href="<%= "showarticle?id=#{article.id}" %>"> <%= "#{article.title}" %></a><br></td>
```

Comment:
is it actually working? you should not use `"` for nesting..... i mean you should have done
`.. href='<%= "showarticle?id=#{article.id}" %>'> ...`